db_schema/schema.sql

```diff
@@ -1,3 +1,19 @@
+-- Open Podcast Analytics database schema
+
+-- migration table to track applied migrations
+CREATE TABLE IF NOT EXISTS migrations (
+  migration_id INTEGER NOT NULL,
+  migration_name VARCHAR(64) NOT NULL,
+  migration_date TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (migration_id)
+);
```

Comment:
I'm confused why this is here if it's also in `migrations/1.sql1`. Can we add an explanation why we need the duplication or was that a mistake?

Reply:
good idea

Reply:
nope, it is on purpose. will make it more clear.

Reply:
see docs in schema.sql, is it clearer now?

Reply:
ah got it. thanks